docs/OPENSHIFT.md

```diff
@@ -0,0 +1,186 @@
+# LogDNA Agent on OpenShift
+
+The agent has been tested on OpenShift 4.4, but should be compatible with any OpenShift cluster packaged with Kubernetes version 1.9 or greater.
+
+## Table of Contents
+
+* [Installing](#installing)
+  * [Installation Prerequisites](#installation-prerequisites)
+  * [Installation Steps](#installation-steps)
+* [Upgrading](#upgrading)
+  * [Upgrading the Configuration](#upgrading-the-configuration)
+    * [Upgrading from Configuration v1.x.x](#upgrading-from-configuration-v1xx)
+  * [Upgrading the Image](#upgrading-the-image)
+* [Uninstalling](#uninstalling)
+* [Run as Non-Root](#run-as-non-root)
+* [Collecting Node Journald Logs](#collecting-node-journald-logs)
+  * [Enabling Journald on the Node](#enabling-journald-on-the-node)
+  * [Enabling Journald Monitoring on the Agent](#enabling-journald-monitoring-on-the-agent)
+
+## Installing
+
+The agent can be effortless installed in your cluster using a set of yamls we provide. These yamls contain the minimum necessary OpenShift Objects and settings to run the agent. Teams should review and modify these yamls for the specific needs of their clusters.
```

Comment:
"can be effortlessly"

Reply:
yeah true... we need to avoid subjective words like "effortless" and "easy" and "fast."

Reply:
Removed!